docs/Developers/Git_Basics.rst

```diff
@@ -159,6 +175,11 @@ shift + Z twice).
 Pushing the commit
 ------------------
 
+Once again, make sure you're on the right branch! If you push from the master
+branch, all you're doing is creating problems for yourself. If you've been
+following this guide, and this is your first pull request, you should
```

Comment:
The reasoning behind this is because the issue said use periodic reminders to make sure people don't mess things up don't the line.